.bumpedrc

```diff
@@ -4,7 +4,7 @@ plugins:
   postrelease:
     Committing new version:
       plugin: bumped-terminal
-      command: 'git add package.json && git commit -m "Release $newVersion"'
+      command: 'git add -f -A package.json dist/ && git commit -m "Release $newVersion"'
```

Comment:
something happened when you pulled these changes locally. but the build stuff should not be part of this PR

Reply:
Yep, not sure what happened there. I'll remove it.